Tarefa 1: Extrair zip e exibir notas fiscais na grid

tainy2017 opened the pull request:
Por favor, analisar se o código atende a primeira tarefa e se já pode ser aprovado. :)

tainy2017 committed:
Alterações: Framework 4.5, criação dos métodos Descompactar Zip, BuscaArquivos, novoObjeto e início da implementação de BtnBuscarNfe.

tainy2017 committed:
Implementação de novoObjeto e buscarArquivo atualizadas. Problemas com a descompactação e leitura dos arquivos.

tainy2017 committed:
*Conseguindo abrir e descompactar zip.
*Atributos de NotaFiscal devidamente setados de acordo com a classe NFE.
*Imprimindo na Grid apenas a última nota fiscal "*.xml" lida da pasta descompactada. Falta resolver esse problema.

tainy2017 committed:
Exibindo notas fiscais na Grid.

tainy2017 committed:
Consegue abrir os arquivos xml da subpasta do zip.

tainy2017 committed:
Extraindo e exibindo na grid. Resolves #1

IgorTerriaga reviewed (changes requested):
Na classe ExtratoImposto, siga o pascal case e coloca o atributo *diferenca* com a inicial maiúscula.
Remova todas as diretivas *using* desnecessárias.
No método DirectorioExiste(), documente o método com /// (3 barras) ficando do que seguinte modo:
/// <summary>
    /// Coloque o que o metodo faz.
    /// </summary>
    /// <param name="nomedoparametro">Descrição do parametro.</param>
    /// <param name="nomedoparametro">Descricao do parametro</param>

tainy2017 committed:
Correções conforme solicitadas:
*Na classe ExtratoImposto, siga o pascal case e coloca o atributo diferenca com a inicial maiúscula.
*Remova todas as diretivas using desnecessárias.
No método DirectorioExiste(), documente o método com /// (3 barras)

tainy2017 commented:
@IgorSantos1996  correções feitas. Verificar novamente, para ontem :) ops, por favor

tainy2017 committed:
Consegue abrir tanto arquivo "*.zip" como apenas um arquivo "*.xml", e exibir na grid.

tainy2017 committed:
Refatoração de nomes de atributos.

tainy2017 committed:
Formatei os campos da Grid.

tainy2017 committed:
Formatando valores numéricos.
